login_V1

```diff
@@ -40,47 +35,100 @@
+					
+					<div class="row" style="display: grid; grid-template-columns: 1fr 1fr; background-color: white; color: darkgreen; border-bottom: 0.1vw solid grey; transition: 0.15s ease-in-out;">
+
+							<div class="ID" style="grid-column: 1;"><p style="margin: 0.5vw 2vw; font-size: 1vw;">Username</p></div>
+
+							<div class="Title" style="grid-column: 2; border-right: 0.1vw solid grey;"><p style="margin: 0.5vw; font-weight: bold; font-size: 1vw;">Password</p></div>
+
+						</div>
+					
+					<?php
+							require "Database_mysqli.php";
+
+							//Creates a varible to store the sql query
+							$query = ("SELECT u.Username, u.Password
+							FROM Users AS u
+							
+							ORDER BY u.Username ASC");
+
+							//Runs and stores the query using the varibles $con (see nav.php) and $query
+							$result = mysqli_query($conn,$query);
+							//runs in a 'while' loop
+							while($output=mysqli_fetch_array($result))
+							{
+							
+						?>
+						<!--php is above. HTML is below. Used to output the query results-->
+
+						<div class="row" style="display: grid; grid-template-columns: 1fr 1fr; border-bottom: 0.1vw solid grey; transition: 0.15s ease-in-out;">
+
+							<div class="ID" style="grid-column: 1;"><p style="margin: 0.5vw 2vw; font-size: 1vw;"><?php echo $output['Username']; ?></p></div>
+
+							<div class="Title" style="grid-column: 2; border-right: 0.1vw solid grey;"><p style="margin: 0.5vw; font-weight: bold; font-size: 1vw;"><?php echo $output['Password']; ?></p></div>
+
+						</div>
+					
+						<?php
+						//Closes the output while loop
+						}
+						?>
+				
+				</div>
+						
 		
 			</div>
 			
-			<div class="section3">
+			<div class="section3" style="z-index: 3; margin-right: 3vw;">
+			
+				<img src="images/icon_1.png" alt="Italian Trulli" style="height: 10vw; width: 10vw; margin: 6vw 0vw 0vw 11vw;" draggable="false">
 				
-				<h3 style="color: darkblue; margin: 6vw 0vw 0vw 13vw; font-size: 2vw; font-weight: bold;">LOGIN</h3>
+				<h3 style="color: black; margin: 0.1vw 0vw 0vw 13vw; font-size: 1.5vw; font-weight: bold;">DELETE</h3>
 			
 				<h3><form method = "post" id = "01_login">
 					<input style="border: none; 
-								border-bottom: 0.25vw solid darkblue;
+								border-bottom: 0.25vw solid #1a1e36;
 								padding: 0.4vw 0.6vw;
 								margin: 2vw 2vw 0vw 4.5vw;
 								width: 21vw;
 								font-size: 1.2vw;"
-					type = "text" name = "username" placeholder="Username"/>
-					
-					<input type = "password" name = "password" placeholder="Password" id = "myInput"/>
+					type = "text" name = "username"  autocomplete="off" placeholder="Username"/>
 					
 					<!-- An element to toggle between password visibility -->
 					
-					<input type="checkbox" onclick="myFunction()"><label for="checkbox" style="font-size: 1vw; font-weight: normal;">Show Password</label>
+					<input type = "submit" value = "Commence Erradication"/>
```

Comment:
I have removed the checkbox from the login page template directly to the submit button in this User Delete Page.